src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -247,16 +253,16 @@ const ScenarioParameters = ({
 
   const processScenarioLaunch = async (forceUpdate) => {
```

Comment:
Like this function, we have a lot that are not a useCallback and used as callback methods of components.
We can maybe use this refactoring feature to change that.